QuickDeployment/QuickDeploymentSetup.md

```diff
@@ -5,113 +5,124 @@ You will receive a download link via email for an archive of the VM image. Once
 
-1. Download OVA file and unzip it to the directory you wish to store it
-2. Review instructions for deploying a VM from an OVA file [here](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.html.hostclient.doc/GUID-FBEED81C-F9D9-4193-BDCC-CC4A60C20A4E_copy.html)
+1. Download OVA file and unzip it to the directory you wish to store it.
+2. Review instructions for deploying a VM from an OVA file [here](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.html.hostclient.doc/GUID-FBEED81C-F9D9-4193-BDCC-CC4A60C20A4E_copy.html).
 3. Adjust settings of newly imported VM to our recommended:
     - 4 vCPUs
     - 8 GB RAM
-4. Install VMWare Tools on the VM once booted (this will require a reboot)
+4. Install VMWare Tools on the VM once booted (this will require a reboot).
 
 ### Platform: VMware (VMDK file)
 
-1. Download VMDK from provided link, and unzip it to the directory you wish to store it
-2. For ESX/ESXi, open vSphere and upload the downloaded VMDK to your datastore
-3. Create a new VM
-4. When prompted for OS type, choose Windows Server 2019 (if available), or Windows 10
-5. If prompted for boot firmware, choose `Legacy BIOS` (**not** UEFI)
-6. When asked to create a new disk or attach, delete the default disk, select attach, and browse to the VMDK you uploaded
+1. Download VMDK from provided link, and unzip it to the directory you wish to store it.
+2. For ESX/ESXi, open vSphere and upload the downloaded VMDK to your datastore.
+3. Create a new VM.
+4. When prompted for OS type, choose Windows Server 2019 (if available), or Windows 10.
+5. If prompted for boot firmware, choose `Legacy BIOS` (**not** UEFI).
+6. When asked to create a new disk or attach, delete the default disk, select attach, and browse to the VMDK you uploaded.
 7. Adjust the hardware specifications of the VM. For a performant system, the following are recommended:
     - 4 vCPUs
     - 8 GB RAM
-8. Install VMWare Tools on the VM once booted (this will require a reboot)
+8. Install VMWare Tools on the VM once booted (this will require a reboot).
 
 Video Summary:
 
 ![QDE VMware VMDK](images/quickdeploy/QDE-vmdk.gif)
 
-
 ## Other Considerations for Virtual Environment
 
-* DNS Settings
-* Set static IP address
-* What if they add it to AD?
+### DNS Settings
+
+The QDE environment is configured by default to use DHCP for easier initial setup.
+You may need to reconfigure it with a static IP address depending on your organization's policies.
 
 ## Virtual Environment Setup
 
-On the desktop of your QDE VM, there is a `Readme.html` file, that will guide you through the rest of the setup process once you are logged in. A version of this readme file can be found [here](QuickDeploymentDesktopReadme.md)
+On the desktop of your QDE VM, there is a `Readme.html` file, that will guide you through the rest of the setup process once you are logged in.
+A version of this readme file can be found [here](QuickDeploymentDesktopReadme.md).
```

Comment:
Change this link, it won't work. "can be found at [[Quick Deployment Desktop Readme|QuickDeploymentDesktopReadme.md]]."

Just FYI it's bad accessibility to link "here" - strive to never do that if possible, or as little as possible. 👍

Reply:
yeah, we can reword that a bit better too. 👍